python/test/t_Viewer.py

```diff
@@ -13,6 +13,14 @@
 
     from openturns.viewer import View
     import openturns as ot
+    import numpy as np
+
+    def BuildSampleFromSequence(sequence):
+        n = len(sequence)
+        data = np.array(sequence)
+        data = data.reshape(n, 1)
+        sample = ot.Sample(data)
+        return sample
```

Comment:
please drop this function, just use `x = [[i] for i in range(n)]`

Reply:
Much simpler indeed. Done.